### PR TITLE
Integrate gpu-manager

### PR DESCRIPTION
**Pull request by taenzeyang**
Signed-off-by: tianze.yang@easystack.cn

### CODE REVIEW

File: assets/state-gpu-admission/0300_daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: DaemonSet
```
Comment (yakiduck) [inline assets/state-gpu-admission/0300_daemonset.yaml]: Deployment，not DaemonSet

File: assets/state-gpu-admission/0300_daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          name: gpu-admission
+          env:
+            - name: LOG_LEVEL
+              value: "FILLED BY THE OPERATOR"
+          ports:
+            - containerPort: 3456
+      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: true
+      priorityClassName: system-cluster-critical
+      nodeSelector:
```
Comment (yakiduck) [inline assets/state-gpu-admission/0300_daemonset.yaml]: fill nodeSelector by Operator from clusterpolicy

File: docker/Dockerfile [hunk 3]
```diff
@@ -70,7 +70,7 @@ ENV VERSION=${VERSION}
 ENV GIT_COMMIT=${GIT_COMMIT}
 
 # Install must-gather dependency: `kubectl`
-RUN OS_ARCH=${TARGETARCH/x86_64/amd64} && OS_ARCH=${OS_ARCH/aarch64/arm64} && curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${OS_ARCH}/kubectl
+COPY ./kubectl ./kubectl
```
Comment (yakiduck) [inline docker/Dockerfile]: revert this line

File: assets/state-gpu-admission/0300_daemonset.yaml (hunk 2 above)
Comment (taenzeyang) [reply assets/state-gpu-admission/0300_daemonset.yaml]: @yakiduck Is this necessary? I found that the nvidia-device-plugin's nodeSelector is not configurable.

File: assets/state-gpu-admission/0300_daemonset.yaml (hunk 1 above)
Comment (taenzeyang) [reply assets/state-gpu-admission/0300_daemonset.yaml]: done by [commit](https://github.com/easystack/gpu-operator/pull/1/commits/c0acab60a35c70b9daaf2837b5871d2f42a25f1d)

File: docker/Dockerfile (hunk 3 above)
Comment (taenzeyang) [reply docker/Dockerfile]: done by [commit](https://github.com/easystack/gpu-operator/pull/1/commits/c0acab60a35c70b9daaf2837b5871d2f42a25f1d)

File: assets/state-gpu-admission/0300_daemonset.yaml (hunk 2 above)
Comment (taenzeyang) [reply assets/state-gpu-admission/0300_daemonset.yaml]: @yakiduck I think it's because the tag's lifecycle can't be managed by the operator

Comment (taenzeyang) [reply assets/state-gpu-admission/0300_daemonset.yaml]: @yakiduck Maybe your configuration management can solve this problem, and it's not too late to do add this feature.

Comment (taenzeyang) [reply assets/state-gpu-admission/0300_daemonset.yaml]: done by [commit](https://github.com/easystack/gpu-operator/pull/1/commits/67e39ed05bf35fb8c98d9118201db2c063cf1b0a)

**Pull request merged**